src/WebJobs.Script.WebHost/Middleware/ScriptHostCheckMiddleware.cs

```diff
@@ -1,9 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
```

Comment:
Is the change here required? Just removing unused usings? Or was there something that got lost in a rebase?

Reply:
obsolete since Mathew replaced this file with another implementation.